src/Features/Core/Portable/ExtractInterface/AbstractExtractInterfaceService.cs

```diff
@@ -270,10 +273,10 @@ public ExtractInterfaceResult ExtractInterfaceFromAnalyzedType(ExtractInterfaceT
             return unformattedInterfaceDocument;
         }
 
-        private static Solution GetSolutionWithFormattedInterfaceDocument(Document unformattedInterfaceDocument, CancellationToken cancellationToken)
+        private static Solution GetSolutionWithFormattedInterfaceDocument(Document unformattedInterfaceDocument, ExtractInterfaceOptionsResult extractInterfaceOptions, CancellationToken cancellationToken)
         {
             Solution solutionWithInterfaceDocument;
```

Comment:
same here. it should either get root or changed to Async method rather than call GetSyntaxRootSynchronously.